newIDE/app/src/AssetStore/AssetDetails.js

```diff
@@ -269,8 +273,7 @@ export const AssetDetails = ({
                       fixedWidth={FIXED_WIDTH}
                     />
                   )}
-                {(asset.objectType === 'tiled' ||
-                  asset.objectType === '9patch') && (
+                {asset.objectType !== 'sprite' && (
```

Comment:
What will happen to users of the 140 if we introduce a new kind of object in the future?

Reply:
Good question.
I believe that an asset always have an array of previewImageUrls defined.
And the CorsAwareImage handles the case where we pass null.
So if we introduce a new kind of object, it should display the preview image correctly (until we push an update where we display that differently)

I think it's fair to say:
- all assets have previews (shown on Asset Search list), and that's the default display, unless specified otherwise